lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/repl.txt

```diff
@@ -0,0 +1,32 @@
+
+{{alias}}( arrays, shapes, fcn )
+    Applies a ternary callback to elements in three broadcasted input arrays and
+    assigns results to elements in a five-dimensional nested output array.
+
+    Parameters
+    ----------
+    arrays: ArrayLikeObject
+        Array-like object containing three input nested arrays and one output
+        nested array.
+
+    shapes: Array<Array<integer>>
+        Array shapes.
+
+    fcn: Function
+        Ternary callback.
+
+    Examples
+    --------
+    > function fcn( x, y, z ) { return x + y + z };
+    > var x = [ 1.0 ];
+    > var y = [ [ 2.0, 2.0 ] , [ 3.0, 3.0 ] ];
+    > var z = [ [ [ [ [ 2.0 ] , [ 3.0 ] ] ] ] ];
+    > var out = [ [ [ [ [ 0.0 ,0.0 ], [ 0.0 ,0.0 ] ] ] ] ];
+    > var shapes =[ [ 1 ], [ 2, 2 ], [ 1, 1, 1, 2, 1 ], [ 1, 1, 1, 2, 2 ] ];
+    > {{alias}}( [ x, y, z, out ], shapes, fcn );
+    > out
+    [ [ [ [ [ 5.0 ,5.0 ], [ 7.0 ,7.0 ] ] ] ] ]
```

Comment:
@MeastroZI Same comment. I suggest doing a line-by-line review of your PR to ensure consistent formatting.